cypress/support/commands.js

```diff
@@ -26,6 +26,10 @@
 
 const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
 
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder = "${placeholder}"]`);
+});
```

Comment:
This custom command `findByPlaceholder` does not return anything. In Cypress, if you want to chain commands off a custom command, you need to ensure that the custom command yields a value. You should return the result of `cy.get()` here.